Farmacy/Models/Crop.h

```diff
@@ -0,0 +1,24 @@
+//
+//  Crop.h
+//  Farmacy
+//
+//  Created by Trang Dang on 7/12/22.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Crop : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSString *name;
```

Comment:
I think it's better to have an auto-incremented primary key, like id. Example https://www.w3schools.com/sql/sql_autoincrement.asp